README.md

```diff
@@ -1243,6 +1243,185 @@ module.exports = {
 };
 ```
 
+### Using `Interoperable CSS` features only
+
+The following setup is an example of allowing `Interoperable CSS` features such as `:import` and `:export` without using modules by setting `compileType` option.
+
+**webpack.config.js**
+
+```js
+module.exports = {
+  module: {
+    rules: [
+      // ...
+      // --------
+      // SCSS
+      {
+        test: /\.scss$/,
+        exclude: /\.module\.scss$/,
+        use: [
+          {
+            loader: 'style-loader'
+          },
+          {
+            loader: 'css-loader',
+            options: {
+              importLoaders: 3,
+              sourceMap: true,
+              modules: {
+                compileType: 'icss'
+              }
+            }
+          },
+          {
+            loader: 'postcss-loader',
+            options: {
+              sourceMap: true
+            }
+          },
+          {
+            loader: 'resolve-url-loader',
+            options: {
+              sourceMap: true
+            }
+          },
```

Comment:
Let's remove unnecessary loaders from here too, example should be simple with only necessary stuff and description

Reply:
see the latest commit, it still has `sourceMap` for loaders. can remove `sourceMap` from all of them if necessary.

Reply:
Yes, let's remove `sourceMap`, because it is not related to this

Reply:
`resolve-url-loader` is also unnecessary for this stuff

Reply:
will do, give me a sec (in real life scenario source maps are kind of necessary, as some of the common loaders that are used in the css related chains break (postcss plugins and what not) 😉 , at least it was so for a very long time, haven't checked recent "default behaviour" for them)

Reply:
done